api/APIv2.md

```diff
@@ -384,25 +384,19 @@ The request MUST use the Device certificate to sign the protectedPayload in the
 
 The request MUST be of mime type "application/x-proto-binary".
 
-The request body MUST be a protobuf message of type AuthContainer where the AuthBody is a protobuf message of type [log.LogBundle](./proto/logs/log.proto). The message itself contains zero, one or more entries of type [log.LogEntry](./proto/logs/log.proto).
+The request body MUST be a protobuf message of type AuthContainer where the AuthBody is a gzip binary message. The binary message is the compressed data and it contains one or more entries, separated by newline '\n' character, of type [log.LogEntry](./go/logs/log.pb.go) in JSON format. Gzip message has the 'header' structure which contains 'Comment', 'Name' strings, and 'ModTime' with golang time.Time format. The 'Comment' field is used to encode 'devID', 'image' and 'eveVersion' of the 'LogBundle' [log.LogBundle](./go/logs/log.pb.go) message in JSON format, and the 'ModTime' field is used for the gzip file creation time. The 'Name' field is not used.
```

Comment:
golang time.Time can be formatted in different ways. Is this a RFC3339 format (which is what we get out of logrus - formated eg as 2020-10-16T08:43:59.748031072Z)? If so it makes sense to say that it is RFC3339 formated timestamp.

Reply:
this read.ModTime is actually not formatted, but assigned directly the golang 'time.Now()' into. So the receiving side can do whatever format it wants. And currently controller side does not actually use this for anything. I'll explicitly say here it's assigned time.Now() for that.